dashboard-for-web/articles/web-dashboard-designer-mode/providing-data/dashboard-data-source-wizard/specify-data-source-settings-json.md

```diff
@@ -1,43 +1,75 @@
 
-Select a new data connection on the "Choose Connection (JSON)" page and click **Next**.
-
 ## Configure a New Data Connection
 
-On the next page, specify the connection name and select the JSON source type:
+On the next page, configure a new data connection:
 
 ![](../../../../images/dashboard-data-source-wizard-configure-data-connections.png)
 
-* **Web Service Endpoint (URI)**
+Specify the connection name and select the JSON source type.
 
-   A URL to a file in the JSON format. You can also specify the Web Service Endpoint's request parameters (username and password, HTTP headers, or query parameters).
+### Web Service Endpoint (URI)
 
-* **JSON String**
+A URL to a file in the JSON format. You can also specify the Web Service Endpoint's request parameters (username and password, HTTP headers, or query parameters).
 
-   A string that contains JSON data. You can also use the Upload JSON button to load content from the selected JSON file.
+![](../../../../images/web-dashboard-json-datasource-parameters.png)
 
-Click **Next** to proceed to the "[Select Data Fields](#select-data-fields)" page.
+- A **path parameter** appends a path element to a JSON endpoint's Uri.
+- A **query parameter** specifies a HTTP request parameter that is passed to a JSON endpoint.
+- A **header** is a custom HTTP header of JSON endpoint requests.
 
-## Select an Existing Data Connection
+You can use expressions to specify path parameters, query parameter values, and headers.
+
+Click the **F** icon  to switch the **Value** option to **Expression Editor** and click **...**  to invoke the editor.
+
+![](../../../../images/web-dashboard-wizard-json-parameters-expression-editor-f-icon.png)
+
+Double-click the expression in the invoked Expression Editor and click **OK**.
 
-If not allowed by your application vendor to create new JSON data connections, the "Choose a data connection" page looks as follow: 
+![](../../../../images/web-dashboard-wizard-json-parameters-expression-editor.png)
 
-![](../../../../images/dashboard-data-source-wizard-choose-data-connectors.png)
+An expression can include [dashboard parameters](../../data-analysis/dashboard-parameters.md). 
 
-Select an existing connection from the list and click **Next** to proceed to the "Select Data Fields" page.
+> [!Tip]
+> Refer to the [Creating Parameters](../../data-analysis/dashboard-parameters/creating-parameters.md) topic for more information how to create a dashboard parameter. 
 
+Select the **Fields** section in the Expression Editor, double-click the predefined dashboard parameter, and click **OK**.
+
+![](../../../../images/web-dashboard-wizard-json-parameters-expression-editor-select-dashboard-parameters.png)
+
+Path parameters and query parameters are included in endpoint requests in the same order as they are listed. Move a parameter up or down in the list to change its position in endpoint requests.
+
+The read-only **Resulting URI** field displays the resulting JSON URI.
```

Comment:
Can you clarify what you mean by "resulting JSON URI"? Is it a newly generated JSON URI (as a result of the above actions)?